src/utils/address_book.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from typing import Dict
+
+from thespian.actors import ActorAddress
+
+from src.utils.constants import ActorNames
+
+
+class AddressBook:
```

Comment:
pls elaborate the purpose of this class, seems like a simple dictionary would suffice, esp if were leaving inter-actor communication to the main actor.

Reply:
Yes, a dictionary is what is being used now, since this object arrives empty when it is passed between threads. Tried to read into it but couldn't find much regarding the cause.